pkg/detector/ospkg/detect.go

```diff
@@ -56,31 +62,31 @@ func (d Detector) Detect(_, osFamily, osName string, _ time.Time, pkgs []ftypes.
 	return vulns, eosl, nil
 }
 
+// nolint: gocyclo
+// TODO: fix cyclometic complexity by removing default
```

Comment:
I'd like to know why `gocyclo` complains `default`. Is it only about `default`? Or, even if we remove `default` and add a new family, will `gocyclo` complain again?

Reply:
even if you replace the default with a different case, it will complain because by default cyclomatic complexity of 10, and for this function, default branch is adding up to 11

Reply:
So, removing `default` doesn't address the root issue. Let's consider it carefully later.